lib/convergence/cli.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'thor'
+require 'convergence/module'
+require 'convergence/config'
+
+class Convergence::CLI < Thor
+  default_command :__fallback # TODO: `__fallback` will be removed in a future version(maybe v0.4.0)
+ 
+  map %w[--version -v] => :version
+
+  desc 'apply FILE', 'execute sql to your database'
+  method_option :config, aliases: '-c', type: :string, required: true, desc: 'Database Yaml Setting'
+  method_option :dry_run, type: :boolean
+  def apply(file)
+    opts = { input: file }
+    if options[:dry_run]
+      require 'convergence/command/dryrun'
+      Convergence::Command::Dryrun.new(opts, config: config).execute
+    else
+      require 'convergence/command/apply'
+      Convergence::Command::Apply.new(opts, config: config).execute
+    end
+  end
+
+  desc 'diff FILE1 FILE2', 'print diff of DSLs'
+  def diff(file1, file2)
+    require 'convergence/command/diff'
+    opts = { diff: [file1, file2] }
+    Convergence::Command::Diff.new(opts, config: config).execute
+  end
+
+  desc 'export', 'export db schema to dsl'
+  method_option :config, aliases: '-c', type: :string, required: true, desc: 'Database Yaml Setting'
+  def export
+    require 'convergence/command/export'
+    opts = {}
+    Convergence::Command::Export.new(opts, config: config).execute
+  end
+
+  desc 'version', 'print the version'
+  def version
+    require 'convergence/version'
+    puts "version #{Convergence::VERSION}"
+  end
+
+  # TODO: `__fallback` will be removed in a future version(maybe v0.4.0)
```

Comment:
Hi, @nishio-dens 

Congratulation on v1.0.0 🎉.

I was thinking next version is v0.3.0 and deprecated codes will be dropped on v0.4.0.
But convergence has been release v1.0.0 now. We should  change this comments.
When do you think drop deprecated code?

Reply:
Oh, sorry. I missed this commend.

Let's drop this code when it becomes version 1.1.0.

Reply:
I fixes on PR #61